Packs/CofenseTriage/Integrations/CofenseTriage/CofenseTriage.py

```diff
@@ -224,28 +301,39 @@ def search_reports_command() -> None:
 
 def search_reports(subject=None, url=None, file_hash=None, reported_at=None, created_at=None, reporter=None,
                    verbose=False, max_matches=30) -> list:
+    # TODO move to new TriageReportQuery (or similar) class
```

Comment:
I think having only the `TriageInstance` and handle the data here is more readable.